pkg/kn/commands/revision_describe.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2018 The Knative Authors
```

Comment:
nit: 2019 :upside_down_face:

Reply:
good catch. gotta fix it all across the board though. probably in a separate PR